index.html

```diff
@@ -195,69 +335,537 @@ <h2>
+      </section>
+      <section>
+        <h3>
+          Constructing a `Gamepad`
+        </h3>
+        <p>
+          <dfn>A new `Gamepad`</dfn> representing a connected gamepad device is
+          constructed by performing the following steps:
+        </p>
+        <ol>
+          <li>Let |gamepad:Gamepad| be a newly created {{Gamepad}} instance:
+            <ol>
+              <li>Initialize |gamepad|'s {{Gamepad/id}} attribute to an
+              identification string for the gamepad.
+              </li>
+              <li>Initialize |gamepad|'s {{Gamepad/index}} attribute to the
+              result of [=selecting an unused gamepad index=] for |gamepad|.
+              </li>
+              <li>Initialize |gamepad|'s {{Gamepad/mapping}} attribute to the
+              result of [=selecting a mapping=] for the gamepad device.
+              </li>
+              <li>Initialize |gamepad|.{{Gamepad/[[connected]]}} to `true`.
+              </li>
+              <li>Initialize |gamepad|.{{Gamepad/[[timestamp]]}} to the
+              [=current high resolution time=].
+              </li>
+              <li>Initialize |gamepad|.{{Gamepad/[[axes]]}} to the result of
+              [=initializing axes=] for |gamepad|.
+              </li>
+              <li>Initialize |gamepad|.{{Gamepad/[[buttons]]}} to the result of
+              [=initializing buttons=] for |gamepad|.
+              </li>
+            </ol>
+          </li>
+          <li>Return |gamepad|.
+          </li>
+        </ol>
+        <p>
+          To <dfn data-lt="selecting an unused gamepad index">select an unused
+          gamepad index</dfn> for |gamepad:Gamepad|, run the following steps:
+        </p>
+        <ol>
+          <li>Let |navigator:Navigator| be |gamepad|'s [=relevant global
+          object=]'s {{Navigator}} object.
+          </li>
+          <li>Let |maxGamepadIndex:long| be the [=list/size=] of
+          |navigator|.{{Navigator/[[gamepads]]}} − 1.
+          </li>
+          <li>[=list/For each=] |gamepadIndex:long| of [=the range=] from 0 to
+          |maxGamepadIndex|:
+            <ol>
+              <li>If |navigator|.{{Navigator/[[gamepads]]}}[|gamepadIndex|] is
+              `null`, then return |gamepadIndex|.
+              </li>
+            </ol>
+          </li>
+          <li>[=list/Append=] `null` to |navigator|.{{Navigator/[[gamepads]]}}.
+          </li>
+          <li>Return the [=list/size=] of
+          |navigator|.{{Navigator/[[gamepads]]}} − 1.
+          </li>
+        </ol>
+        <p>
+          To <dfn data-lt="initializing axes">initialize axes</dfn> for
+          |gamepad:Gamepad|, run the following steps:
+        </p>
+        <ol>
+          <li>Let |inputCount:long| be the number of axis inputs exposed by the
+          device represented by |gamepad|.
+          </li>
+          <li>Set |gamepad|.{{Gamepad/[[axisMinimums]]}} to a [=list=] of
+          `unsigned long` values with [=list/size=] equal to |inputCount|
```

Comment:
```suggestion
          {{unsigned long}} values with [=list/size=] equal to |inputCount|
```

Plus a few other ones...